Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveHM_2nd.cpp

```diff
@@ -153,6 +158,31 @@ void FiniteDifferenceSolver::MacroscopicEvolveHMCartesian_2nd(
                         Hz_eff += MacroscopicProperties::face_avg_to_face(i, j, k, 0, amrex::IntVect(0, 0, 1), amrex::IntVect(1, 0, 0), Hz_old);
                     }
 
+                    int mag_exchange = 0;
+                    if (mag_exchange == 1){
+                        // H_exchange
+                        amrex::Real M_exchange_constant = 0.001;
+                        Hx_eff += 2.0 * M_exchange_constant / PhysConst::mu0 / mag_Ms_arrx / mag_Ms_arrx * T_Algo::Laplacian(M_xface, coefs_x, coefs_y, coefs_z, i, j, k, 0);
+                        Hy_eff += 2.0 * M_exchange_constant / PhysConst::mu0 / mag_Ms_arrx / mag_Ms_arrx * T_Algo::Laplacian(M_xface, coefs_x, coefs_y, coefs_z, i, j, k, 1);
+                        Hz_eff += 2.0 * M_exchange_constant / PhysConst::mu0 / mag_Ms_arrx / mag_Ms_arrx * T_Algo::Laplacian(M_xface, coefs_x, coefs_y, coefs_z, i, j, k, 2);
+                    }                    
+
+
+                    int mag_anisotropy = 0;
+                    if (mag_anisotropy == 1){
```

Comment:
We will ultimately want to test to make sure the anisotropy_constant !=0 and M_s != 0

Reply:
same comment as exchange